src/Datadog.Trace.ClrProfiler.Native/integration.h

```diff
@@ -128,7 +128,7 @@ struct MethodSignature {
   const std::vector<BYTE> data;
 
   MethodSignature() {}
-  MethodSignature(const std::vector<BYTE>& data) : data(data) {}
+  MethodSignature(const std::vector<BYTE> data) : data(data) {}
```

Comment:
If I remember my C++ correctly, removing the `&` here will create an additional copy of the `vector<BYTE>` instead if passing it by reference. Not a big deal, but also no reason to remove it if it's already there.